dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/loadbalance/ShortestResponseLoadBalance.java

```diff
@@ -36,6 +42,46 @@
 
     public static final String NAME = "shortestresponse";
 
+    private static final int SLIDE_PERIOD = Integer.parseInt(System.getProperty("ShortestResponseLoadBalance.slidePeriod", "30000"));
+
+    private ConcurrentMap<RpcStatus, SlideWindowData> methodMap = new ConcurrentHashMap<>();
+
+    private AtomicBoolean onResetSlideWindow = new AtomicBoolean(false);
+
+    private volatile long lastUpdateTime = System.currentTimeMillis();
+
+    protected static class SlideWindowData {
+        private final static ExecutorService EXECUTOR_SERVICE = Executors.newSingleThreadExecutor();
```

Comment:
add ThreadFactory for this executor

Reply:
Thanks for your advice. I have fix these problems.